orbit/pkg/migration/reader.go

```diff
@@ -0,0 +1,43 @@
+package migration
```

Comment:
do we need a separate reader? context: for the token file, we wanted to semantically enforce that the file could only be read by Fleet Desktop

Reply:
Ahh gotcha! yeah I originally didn't have this, but added it to follow the token pattern. I agree, we probably don't need a separate type here